<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelfCheckItem</class>
 <widget class="QWidget" name="SelfCheckItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="selfCheckContent" native="true">
     <property name="styleSheet">
      <string notr="true">#checkIcon{
	background-image: url(:/public/selfCheck/self_check_not.png);
}
#checkItemName{
	font-size:36px;
	color:#959a9b;
}
#checkErrorCode{
	font-size:36px;
	color:#e27053;
}
#checkJumpOver{
	max-height:0px;
	min-height:0px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0,10,0,5,5" columnminimumwidth="48,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="checkIcon">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-repeat:no;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="checkErrorCode">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>001</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="checkItemName">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>透析液过滤器</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="checkJumpOver">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#checkJumpOver{
	font-size:24px;
	color:#fff;
	border-image: url(:/public/bottom/button_open.png);
}
#checkJumpOver:pressed{
	border-image:url(:/public/bottom/button_ing.png);
}</string>
        </property>
        <property name="text">
         <string>跳过此项</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
